readme.md

```diff
@@ -23,7 +23,7 @@ WP Job Manager is a **lightweight** job listing plugin for adding job-board like
 * Each listing can be tied to an email or website address so that job seekers can apply to the jobs.
 * Searches also display RSS links to allow job seekers to be alerted to new jobs matching their search.
 * Allow logged in employers to view, edit, mark filled, or delete their active job listings.
-* Developer friendly code — Custom Post Types, endpoints & template files.
+* Developer friendly code â Custom Post Types, endpoints & template files.
```

Comment:
What's up with that funk character?

Reply:
Fantastic question. Fixed in https://github.com/Automattic/WP-Job-Manager/pull/2018/commits/edae4a1176362a5bc7ce56d7dabdb6099783391b